.gitignore

```diff
@@ -107,6 +107,7 @@ coverage.xml
 *.cover
 .hypothesis/
 .pytest_cache/
+infra/scripts/*.conf
```

Comment:
Sorry, why is this being added?

Reply:
My script for creating a cluster generates conf files as per redis configuration. ./create-cluster.sh clean should remove all of them but just in case people forget, I just added this line to make sure it's not tracked by git.